docs/setup/kubeedge_install.md

```diff
@@ -0,0 +1,74 @@
+# Install KubeEdge
+
+In this section, we would cover the below topics
+
+1. [Abstract](#Abstract)
+2. [Pre-requisties](#prerequisties)
+3. [Setup KubeEdge](#setup-kubeEdge)
+4. [Configure KubeEdge](#configure-kubeEdge)
+5. [Run KubeEdge](#run-kubeedge)
+6. [KubeEdge Pre-Check](#kubeedge-pre-check)
+## Abstract
+KubeEdge is composed of cloud and edge parts. It is built upon Kubernetes and provides core infrastructure support for networking, application deployment and metadata synchronization between cloud and edge. So if we want to setup KubeEdge, we need to setup Kubernetes cluster, cloud side and edge side.
+
++ on `cloud side`, we need to install Docker, Kubernetes cluster and cloudcore.
++ on `edge side`, we need to install Docker, mqtt and edgecore.
+
+## Prerequisites
+
++ **Go dependency** and **Kubernetes compatibility** please refer to [compatibility-matrix](https://github.com/kubeedge/kubeedge#compatibility-matrix). Please refer this to understand what version of Docker, Kubernetes, Golang can be installed.
+
+### Cloud side (KubeEdge Master Node)
```

Comment:
Please refer above. Cloud side and Cloud part and different variations are used across the docs. This can be confusing. Please define one nomenclature and use it consistently.